tests/test_model.py

```diff
@@ -1202,7 +1325,35 @@ def test_bad_explicit_model_implicit_data(train_val_implicit_sample_data):
     model = MatrixFactorizationModel(train=train,
                                      val=val,
                                      loss='mse')
-    trainer = CollieTrainer(model=model, logger=False, checkpoint_callback=False, max_epochs=1)
+    trainer = CollieTrainer(model=model, logger=False, enable_checkpointing=False, max_epochs=1)
 
     with pytest.raises(ValueError):
         trainer.fit(model)
+
+
+def test_get_item_preds_err(implicit_model,
+                            train_val_implicit_data):
+    model = implicit_model
+    train, val = train_val_implicit_data
```

Comment:
If we end up not using `val` here in the test, I think it's best to be explicit about that with something like: 
```suggestion
    train, _ = train_val_implicit_data
```